src/app/config/navigation.ts

```diff
@@ -0,0 +1,18 @@
+import { WHITE } from '@constants/colors';
+import NavBackground from '@app/components/NavBackground';
+import { NAV_HEIGHT, NOTCHSIZE } from '@constants/commonStyles';
+
+export const stackNavigatorConfig = {
+  headerTitleStyle: {
+    fontSize: 16,
+    color: WHITE,
+    fontWeight: '600',
+    letterSpacing: -0.4
+  },
+  headerStyle: NAV_HEIGHT,
+  headerStatusBarHeight: 10 + NOTCHSIZE,
```

Comment:
Why this?

Reply:
used it to adjust the spacing between the top and the title, I should probably use the headerTitleStyle prop right?

Reply:
Yes, try if it works correctly by moving this style to `headerTitleStyle`

Reply:
done 👍